lib/preload/ndb/preload.js

```diff
@@ -25,14 +25,16 @@
     process.env.NDD_PPID = process.pid;
     if (!process.env.NDD_DATA)
       process.env.NDD_DATA = process.pid + '_ndbId';
-    process.versions['ndb'] = process.env.NDB_VERSION;
+    process.versions['ndb'] = '1.1.0';
+    const inspector = require('inspector');
```

Comment:
nit: There seem to be some unrelated changes. Consider splitting into several commits or amending the commit message.

Reply:
done.